tests/testing/clients/test_django.py

```
@@ -0,0 +1,157 @@
+
+    assert response.status_code == 400
+    assert response.data == b"bad request"
+
+
+@pytest.mark.asyncio
+async def test_async_do_request_handles_streaming_response(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    class FakeStreamingResponse:
+        status_code = 200
+        headers = {"Content-Type": "text/plain"}
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)